setup.py

```diff
@@ -24,15 +24,6 @@
           "PyTango>=9.2.2",
           "numpy",
           "jsonschema"],
-      tests_require=[
-          'katcp',
-          'numpy',
-          'nose_xunitmp'],
-      extras_require={
-          'docs': ["sphinx-pypi-upload",
-                   "numpydoc",
-                   "Sphinx"],
-          },
```

Comment:
Why was this removed?

Reply:
This is covered in the `tox.ini` file under the dependencies

Reply:
Added these to the tox file that was later added after your comment thanks.

Reply:
Are you proposing that tox is used for document generation, but not for testing?

Reply:
As per agreement with @mmphego and @tockards we first wanted your take on making an exception for `tango-simlib` not to use tox, and after that we will refactor everything to be what they need to be.

Reply:
Yes, it is OK for to leave out tox for tango-simlib.

Reply:
Please restore these options on setup.py